automation/README.md

```diff
@@ -9,35 +9,39 @@ See [firecloud-automated-testing](https://github.com/broadinstitute/firecloud-au
 
 ### Set Up
 
-Render configs:
+#### Render configs
+
+##### Against dev fiab Sam
+```bash
+./render-local-env.sh
+```
+
+##### Against local UI pointing to dev fiab Sam
 ```bash
-./render-local-env.sh [branch of firecloud-automated-testing] [vault token] [env] [service root]
+LOCAL_UI=true ./render-local-env.sh
+```
+
+##### Against local Sam pointing to the dev live env
+Run `./render-local-env.sh` and then update `samApiUrl` in application.conf to:
+```
+  samApiUrl = "https://local.broadinstitute.org:50443/"
 ```
 
-**Arguments:** (arguments are positional)
-
-* branch of firecloud-automated-testing
-    * Configs branch; defaults to `master`
-* Vault auth token
-	* Defaults to reading it from the .vault-token via `$(cat ~/.vault-token)`.
-* env
-	* Environment of your FiaB; defaults to `dev`
-* service root
-    * the name of your local clone of sam if not `sam`
-	
 ### Run tests
+All test code lives in `automation/src/test/scala`.
 
 #### From IntelliJ
+It is recommended to `Open...` IntelliJ to the `automation` folder directly and import from SBT.
 
-First, you need to set some default VM parameters for ScalaTest run configurations. In IntelliJ, go to `Run` > `Edit Configurations...`, select `ScalaTest` under `Defaults`, and add these VM parameters:
+First, you need to set some default VM parameters for ScalaTest run configurations. In IntelliJ, go to `Run` > `Edit Configurations...`, select `ScalaTest` under `Templates`, and add these VM parameters:
```

Comment:
These instructions I think are better than the wording in the other README for how to modify the template